lib/appoptics_apm/inst/mongo.rb

```diff
@@ -104,6 +105,7 @@ class Cursor
 
             AppOpticsAPM::API.trace(:mongo, report_kvs) do
               send("#{m}_without_appoptics", *args)
+              report_kvs[:Backtrace] = AppOpticsAPM::API.backtrace if AppOpticsAPM::Config[:mongo][:collect_backtraces]
```

Comment:
why not set `:Backtrace` before the `trace` block? I'm not sure what the `trace()` is doing so this might be a dumb question.